src/register-org/components/organisation-services-access/organisation-services-access.component.html

```diff
@@ -6,14 +6,12 @@
       <span class="govuk-caption-xl">Register an organisation</span>
       <form [formGroup]="servicesFormGroup">
         <div class="govuk-form-group">
-          <fieldset class="govuk-fieldset" aria-describedby="changed-name-hint">
-            <legend class="govuk-fieldset__legend govuk-fieldset__legend--l">
-              <h1 class="govuk-fieldset__heading">
-                Which services will your organisation need to access?
-              </h1>
+          <fieldset class="govuk-fieldset">
+            <legend class="govuk-fieldset__legend govuk-fieldset__legend--xl govuk-!-margin-bottom-8">
+              Which services will your organisation need to access?
             </legend>
-            <hr>
-            <div class="govuk-label govuk-!-font-weight-bold">
+            <hr class="govuk-!-margin-top-0">
+            <div class="govuk-label govuk-!-font-weight-bold govuk-!-margin-top-4">
               Please indicate the services you are looking to access
             </div>
             <div class="govuk-hint">
```

Comment:
@johnbenjamin-hmcts - for accessibility, add `id="service-selection-hint"` and then reference that on Line 9

Reply:
Thank you @Daniel-Lam - I have updated the code now.